site/en/tutorials/load_data/images.ipynb

```diff
@@ -0,0 +1,1356 @@
+        "ds"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "rdxpvQ7VEo3y",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "timeit(ds)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "usIv7MqqZQps",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "One disadvantage to using an im memory cache is that the cache must be rebuilt on each run, giving the same startup delay each time the dataset is started:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "eKX6ergKb_xd",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "timeit(ds)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "jUzpG4lYNkN-",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "If the data doesn't fit in memory, use a cache file:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "vIvF8K4GMq0g",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "ds = image_label_ds.cache(filename='./cache.tf-data')\n",
+        "ds = ds.repeat().shuffle(buffer_size=4000)\n",
+        "ds = ds.batch(BATCH_SIZE).prefetch(1)\n",
+        "ds"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "eTIj6IOmM4yA",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "timeit(ds)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "qqo3dyB0Z4t2",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "The cache file also has the advantage that it can be used to quickly restart the dataset without rebuilding the cache. Note how much faster it is the second time:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "hZhVdR8MbaUj",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "timeit(ds)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "WqOVlf8tFrDU",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "### TFRecord File"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "y1llOTwWFzmR",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "TFRecord files are a simple format to store a sequence of binary blobs. In this case the fact that these are read linearly off disk, which may give some performance boost.\n",
```

Comment:
The second sentence doesn't make much sense to me. Perhaps something like "By packing multiple examples into the same file, TensorFlow is able to read multiple examples at once, which is especially important for performance when using a remote storage service such as GCS."?

Reply:
SGTM